src/digest/sha1.cr

```diff
@@ -11,8 +11,8 @@ class Digest::SHA1 < Digest::Base
   # but we use loop unrolling for faster execution (about 1.07x slower than OpenSSL::SHA1).
 
   @intermediate_hash = uninitialized UInt32[5]
-  @length_low = 0_u32
-  @length_high = 0_u32
+  @length_low : UInt32 = 0_u32
+  @length_high : UInt32 = 0_u32
```

Comment:
No need to add explicit type declaration.

Reply:
👍 
The diff should only be what I mentioned in the forum post (the two lines below).

Reply:
There is a forum post?